manifests/a/Aurora/Aurora/v0.8.1/Aurora.Aurora.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Aurora.Aurora
+PackageVersion: v0.8.1
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x86
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```

```suggestion
  Architecture: x64
```
From the error, I'm guessing that this is a 64-bit application.